### DISCUSSION TITLE
How do I update versions in versions\<letter>\<port>.json and versions\baseline.json

### DISCUSSION THREAD

**Discussion by voltagex** [microsoft/vcpkg#26304] [Q&A] [answered]
The versions & baseline files look generated, how am I supposed to update these if I'm updating a port?

**Comment by Osyotr** [comment, marked as answer]
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-files-in-versions-of-any-modified-ports
